core/app/models/mno_enterprise/base_resource.rb

```diff
@@ -61,16 +61,16 @@ def update_attribute(name, value)
     # emulate active record call of callbacks
     def save(*args)
       run_callbacks :save do
-        super()
+        run_callbacks :update do
+          super()
+        end
       end
     end
 
     # emulate active record call of callbacks, a bit different as before_update is called before before_save
     def update_attributes(attrs = {})
       self.attributes = attrs
```

Comment:
We should consider including `ActiveModel::AttributeAssignment` and use the `assign_attributes` method.
https://github.com/rails/rails/blob/master/activemodel/lib/active_model/attribute_assignment.rb

The current method replaces all the attributes and does not use the model custom setters. Not sure it's ideal.

See original implementation of `update_attributes/update` in `ActiveRecord::Persistence`:
https://github.com/rails/rails/blob/310918f6a194bf5752fe1025930881756f5d8a8e/activerecord/lib/active_record/persistence.rb#L280